### ISSUE TITLE
TypeScript declarations prevent `dhtmlx-gantt` from being imported as a module

### ISSUE THREAD

**Issue by tylerjwatson** [DHTMLX/gantt#38]
Hi,

The `declare module` statements in the TypeScript declaration files prevent it from being imported as an ES module, as `gantt` and `Gantt` are not installable modules from npm.

The declaration statements should be updated to reflect the names of the packages from where they are installed.  I can submit a PR with these changes.

```ts
declare module "gantt" {
    export = gantt;
}
```
should read
```ts
declare module "dhtmlx-gantt" {
    export = gantt;
}
```

as `dhtmlx-gantt` is the name of the installable module.

If Gantt Enterprise is an installable module, then perhaps it should be distributed under its own `dhtmlx-gantt-enterprise` module.  Typing declarations for modules should not have more than one ambient module declaration in it.

Cheers! 👍

**Comment by nrakochy** [comment]
I just installed the module and am facing the same issue. Updated the types file with 
```typescript
declare var gantt: GanttStatic;
export { gantt };
```
to fix the issue, but do not want to modify a library that I don't control. Thanks.

**Comment by gearcoded** [comment]
Hello,
We are aware of that issue. It will be fixed in the future, but I cannot give you any ETA.

**Comment by grantsru** [comment]
I commented on a PR that fixes this issue; considering how big the impact this small change provides to the module, could this be prioritized for the next release, please?

**Comment by gearcoded** [comment]
Hello,
I reminded the dev team about the bug. They will try to include it in the nearest release, but I cannot promise that.

**Comment by gearcoded** [comment]
Hello,
The dev team fixed the bug when Gantt couldn't be imported as an es6 module. You can read more about features and bug fixes in the 7.0 version in the following article:
https://docs.dhtmlx.com/gantt/whatsnew.html#x4465x70x4466x
